### DISCUSSION TITLE
Trying to unravel the mystery of why images won't display and text (in order to be formatted)  must be wrapped in html when calling in from a refinablestring on a custom layout on Modern Search.

### DISCUSSION THREAD

**Discussion by tedtninety** [microsoft-search/pnp-modern-search#3159] [Q&A]
Whew! : Long title - sounds like a Netflix documentary.

This one has been eluding me a while.

**Check list:**

- Column type: multi-text
- Text-type:: Enhanced rich text
- Map  RefinableString
- Call the item in Custom Layout with triple staches {{{slot item @root.slots..definition}}}

So oddly enough.
The list item shows the content ( as seen on the image).
Formatting created using the inline editor is totally ignored.
HTML appended is honored, but the "img" tag is completely ignored.

I am not sure what I am missing.

Are images within a mutlitext - enhanced rich column supposed to show or is that something that  isn't supported?

Is inline SP list text format supported, or am I to use HTML tagging on  text to display it formatted via Modern Search.
 
Thank you.

Please see image for list item in edit mode.
From the SP List Modern Search reads from:
![off232](https://github.com/microsoft-search/pnp-modern-search/assets/115969035/2dd2e4ce-a8ac-4534-a0ba-1b4583ffc930)

Output via Modern Search:
![outpot1](https://github.com/microsoft-search/pnp-modern-search/assets/115969035/7ce518a0-48e6-4fd2-b9fb-12d6e71ac59b)

**Comment by wobba** [comment]
The web parts have added html/script sanitisation to avoid injection attacks. That said, if you share the data coming back on the JSON response it's easier to see if the data coming back is correct, and then stripped, or if the data is not coming back complete as you expect.

And you have to use `{{{ }}}` to have html render as html as you mention.
